app/modules/ads/targeting.js

```diff
@@ -208,4 +208,40 @@ export default {
 
     return targeting;
   },
+
+  getBiddersPrices(slotName) {
+    const { bidders } = window.Wikia.adBidders;
+
+    const realSlotPrices = bidders.getDfpSlotPrices(slotName);
+    const currentSlotPrices = bidders.getCurrentSlotPrices(slotName);
+
+    function transformBidderPrice(bidderName) {
+      if (realSlotPrices && realSlotPrices[bidderName]) {
+        return realSlotPrices[bidderName];
+      }
+
+      if (currentSlotPrices && currentSlotPrices[bidderName]) {
+        return `${currentSlotPrices[bidderName]}not_used`;
+      }
+
+      return '';
+    }
+
+    return {
+      bidder_1: transformBidderPrice('indexExchange'),
+      bidder_2: transformBidderPrice('appnexus'),
```

Comment:
Out of curiosity, why are some of bidder numbers not used?

Reply:
Those are a holes in which now removed bidders was in the past.